projects/core/src/cms/occ/occ-cms-component.adapter.ts

```diff
@@ -76,13 +122,19 @@ export class OccCmsComponentAdapter implements CmsComponentAdapter {
   }
 
   private getComponentsRequestParams(
```

Comment:
The names of `getComponentsRequestParams` and `getComponentRequestParams` are very confusing. I suggest, we rename `getComponentsRequestParams` to `getPaginationParams` and `getComponentRequestParams` to `getContextParams`. So in "get" functions, you call `getPaginationParams` and `getContextParams`, and make the id list string. In "post" function, only need to call `getPaginationParams` and `getContextParams`.